client/src/components/NeighborhoodMap.jsx

```diff
@@ -1,34 +1,47 @@
 import React from 'react';
 import Details from './Details';
 import { Map, GoogleApiWrapper } from 'google-maps-react';
-const googleMapApi = require('../../../googleMapConfig');
+import '../../dist/index.css';
+// DO NOT COMMIT ME TO VERSION CONTROL!!!!!
+const GOOGLE_MAPS_API_KEY = 'AIzaSyALnk5uTIhLwMzcanWu3Dspzuh3Gt47C2k';
 
 class NeighborHoodMap extends React.Component {
   constructor(props) {
     super(props);
-
   }
 
-
   render() {
     const mapStyles = {
-      width: '50%',
-      height: '50%',
+      gridRow: 1/2,
+      width: '58vw',
+      height: '500px',
+      // position: 'relative',
     };
 
     const { lat, lng } = this.props;
```

Comment:
Now that we're going to refactor to functional stateless, let's do this destructuring in the functions input parameters.